src/main/resources/assets/minecraft/lang/nl_nl.json

```diff
@@ -0,0 +1,60 @@
+{
+  "title.multiplayer.other2": "Meerspeler (%s)",
+
+  "foxclient.gui.button.reconnect": "Maak opnieuw verbinding",
+  "foxclient.gui.button.close": "Sluiten",
+
+  "foxclient.gui.toast.title": "FoxClient",
+  "foxclient.gui.toast.modmenu.missing": "ModMenu is niet geïnstalleerd",
+
+  "category.foxclient.main": "FoxClient",
+  "key.foxclient.toggle_hud": "HUD in/uit schakelen",
+  "key.foxclient.freelook": "Freelook",
+  "key.foxclient.configKey": "FoxClient Configuratie",
+  "key.foxclient.zoom": "Zoom",
+
+  "foxclient.rpc.state.idle": "Inactief",
+  "foxclient.rpc.state.singleplayer": "Speelt singleplayer",
+  "foxclient.rpc.state.lan": "Speelt in een LAN-game",
+  "foxclient.rpc.state.multiplayer": "Speelt op %s",
+  "foxclient.rpc.state.multiplayer.count": "Speelt op %s (%s speler)",
+  "foxclient.rpc.state.multiplayer.count.single": "Speelt op %s (%s spelers)",
+  "foxclient.rpc.state.multiplayer.hide_ip": "Speelt multiplayer",
+  "foxclient.rpc.state.realms": "Speelt op een Realm",
+
+  "foxclient.config.category.ingame-hud": "In-Game HUD",
+  "foxclient.config.category.armor-hud": "Pantser HUD",
+  "foxclient.config.category.client": "Cliënt",
+  "foxclient.config.category.eastereggs": "EasterEggs",
+  "foxclient.config.category.menus": "Aangepaste menu's",
+  "foxclient.config.category.misc": "Overige",
+
+  "foxclient.config.smoothzoom": "Smooth Zoom in/uit schakelen",
+
+  "foxclient.config.hudenabled" : "In-Game HUD in/uit schakelen",
+  "foxclient.config.hudping": "Ping",
+  "foxclient.config.hudtps": "TPS",
+  "foxclient.config.hudfps": "FPS",
+  "foxclient.config.hudversion": "Versie",
+  "foxclient.config.hudcoordinates": "XYZ",
+  "foxclient.config.hudbiome": "Bioom",
+  "foxclient.config.hudcoordinatescolor": "Gekleurde XYZ",
+  "foxclient.config.hudserverip": "Server IP",
+  "foxclient.config.hudlogo": "Logo",
+  "foxclient.config.hudbackground": "Achtergrond",
+
+  "foxclient.config.armorhudenabled": "Pantser HUD in/uit schakelen",
+  "foxclient.config.armorhuddisplaypercentage": "Weergeven als percentage",
+
+  "foxclient.config.uwufy": "UwUfy",
+
+  "foxclient.config.custommainmenu": "Hoofdmenu",
+  "foxclient.config.custompausemenu": "Pauzemenu",
+
+  "foxclient.config.discordenabled": "DiscordRPC - in/uit schakelen",
+  "foxclient.config.discordshowip": "DiscordRPC - IP weergeven",
+  "foxclient.config.discordshowplayer": "DiscordRPC - toon eigen speler",
+
+  "foxclient.config.toggle.on": "AAN",
+  "foxclient.config.toggle.off": "UIT"
+}
```

Comment:
i don't see any racial slurs in here but I'd want a dutch person to look over this.
for now this look good enough to be merged though

Reply:
i mean, gangstacat looked over it

Reply:
alright!